doc/rst/users-guide/datapar/forall.rst

```diff
@@ -75,7 +75,7 @@ avoiding overwhelming the target system with too much concurrency.
 
 This is precisely what forall-loops are for.  While they result in
 parallel execution like coforalls, they typically use a number of
-tasks that is appropriate for the system, dividing the loop's
+tasks that are appropriate for the system, dividing the loop's
```

Comment:
In this case also, the sentence was correct as written.  The subject of this verb is "that", referring to "a number", which is singular.